airtable.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const Airtable = require('airtable');
+
+const base = new Airtable({apiKey: process.env.AIRTABLE_API_KEY}).base('appuZMt69pZnTis2t');
+
+const xdContent = {};
+const cacheFilePath = './airtable-cache.json';
+const newsFilePath = './collections/_import/news.md';
+const biosFilePath = './collections/_import/bios.md';
+
+// Utility function we'll use to compare our data
+const deepCompare = (arg1, arg2) => {
+    if (Object.prototype.toString.call(arg1) === Object.prototype.toString.call(arg2)){
+      if (Object.prototype.toString.call(arg1) === '[object Object]' || Object.prototype.toString.call(arg1) === '[object Array]' ){

```

Comment:
Here we can use the `toString` method directly for the first part of the conditional, and `Array.isArray` for the second part:
```suggestion
      if (arg1.toString() === '[object Object]' || Array.isArray(arg1)){

```